### PR TITLE
Rchain 2843 Integration tests: Implement test_validator_catching_up

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
Integration tests: Implement test_validator_catching_up

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-2843

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by zsluedem** [comment]
@adaszko Done. Have a look and delegate, please

File: integration-tests/test/test_internal.py [hunk 1]
```diff
@@ -87,6 +89,54 @@ def test_parse_show_blocks_output() -> None:
...
...b)} ) {\n      @{Unforgeable(0x39bdf789fb12e1c3487abce560193e3a9545d2d7e311aca70f6f47d36ba5115a)}!((x9).size(), x8, *x5) |\n      for( @{x10} <- x5 ) {\n        x6!(x9, x10, {}, *x7) |\n        for( @{x11} <- x7 ) {\n          @{x4}!((true, \"Bond successful!\")) |\n          @{Unforgeable(0x4de349932e7e2443526ad0a4923cd69c0027184659a6c475112fecca2008dedb)}!((x11).set(x0,((x2 - x8), x1, x3, ((x9).size() + 1))))\n        }\n      }\n    } |\n    for( @{x8}, @{x9}, @{x10}, @{x11} <= x6 ) {\n      match x8 {\n        {x13 : (x14, x15, x16, x17)...x12} => x6!(x12, x9, (x10).set(x13,((x14 + (x9).nth((x17 - 1))), x15, x16, x17)), x11) ;\n        _ => @{x11}!(x10)\n      }\n    }\n  }\n} |\nfor( @{x0}, @{x1}, @{x2} <= @{Unforgeable(0x39bdf789fb12e1c3487abce560193e3a9545d2d7e311aca70f6f47d36ba5115a)} ) {\n  new x3, x4, x5, x6, x7, x8 in {\n    @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"range\", 1, (x0 + 1), *x3) |\n    for( @{x9} <- x3 ) {\n      @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"map\", x9, *x4, *x5) |\n      for( @{x10} <- x5 ) {\n        @{bundle+ {   Unforgeable(0x6
```
Comment (adaszko) [inline integration-tests/test/test_internal.py]: Whoa, that's comprehensive! I like it! 👍 😄

File: integration-tests/test/test_genesis_ceremony.py [hunk 2]
```diff
@@ -77,7 +79,7 @@ def test_successful_genesis_ceremony(command_line_options: CommandLineOptions, r
                         assert validator_a_genesis_block['blockHash'] == ceremony_master_genesis_block['blockHash']
                         assert validator_a_genesis_block['mainParentHash'] == '""'
 
-                        validator_b_blocks = validator_a.show_blocks_parsed(2)
+                        validator_b_blocks = validator_b.show_blocks_parsed(2)
```
Comment (adaszko) [inline integration-tests/test/test_genesis_ceremony.py]: Good catch 👍

**Comment by adaszko** [comment]
bors delegate+

**Comment by zsluedem** [comment]
bors try

**Comment by zsluedem** [comment]
bors try

**Comment by woky** [comment]
bors try

**Comment by woky** [comment]
bors help

**Pull request merged**